tests/core/mining-module/test_miner_setGasPrice.py

```diff
@@ -10,15 +11,35 @@
 
 
 @flaky(max_runs=3)
-def test_miner_setGasPrice(web3_empty, wait_for_block):
+def test_miner_set_gas_price(web3_empty, wait_for_block):
     web3 = web3_empty
 
     initial_gas_price = web3.eth.gasPrice
 
     # sanity check
     assert web3.eth.gasPrice > 1000
 
-    web3.geth.miner.setGasPrice(initial_gas_price // 2)
+    web3.geth.miner.set_gas_price(initial_gas_price // 2)
+
+    with Timeout(60) as timeout:
+        while web3.eth.gasPrice == initial_gas_price:
```

Comment:
even tough I'm pretty sure these are also getting skipped, let's change `gasPrice` to `gas_price`
```suggestion
        while web3.eth.gas_price == initial_gas_price:
```

Reply:
Not sure I should be doing this in this PR, the property `gasPrice` is from the 'eth' module. I'm happy to do the eth module later..

Reply:
Ha, you're right. My bad!